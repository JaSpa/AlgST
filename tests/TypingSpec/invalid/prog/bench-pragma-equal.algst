{-# BENCHMARK! Int Int #-}

{-# BENCHMARK!
dual ?Int.End!
!Int.End?
#-}
