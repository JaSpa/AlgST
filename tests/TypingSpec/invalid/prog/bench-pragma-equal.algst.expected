
1:16: error:
	 Types in benchmark are equal.

4:1: error:
	 Types in benchmark are equal.